skt/kernelbuilder.py

```diff
@@ -41,6 +41,7 @@ def __init__(self, source_dir, basecfg, cfgtype=None,
         self.make_argv_base = ["make", "-C", self.source_dir]
         self.enable_debuginfo = enable_debuginfo
         self.build_arch = self.__get_build_arch()
+        self.cross_compiler_prefix = self.__get_cross_compiler_prefix()
```

Comment:
`os.environ.get('CROSS_COMPILE', None)` :man_shrugging:

Reply:
True. Do you mean that it would have been better to write that instead of having that function?

Reply:
Exactly, less code to maintain, less documentation, etc hehehe. Lazy programmer :man_shrugging:

Reply:
Agreed. Would you care to submit a fix?